server/src/routes/api/v1/dogParksRouter.js

```diff
@@ -44,4 +43,25 @@ dogParksRouter.get("/:id", async (req, res) => {
     }
 })
 
+dogParksRouter.post("/:id", async (req, res) => {  
+    try {
+        const { body } = req
+        const userId = req.user.id
+        body.review.userId = userId
+        const { id } = req.params
+        body.review.dogParkId = id
+        const cleanedInput = cleanUserInput(body.review)
+        const review = await Review.query().insertAndFetch(cleanedInput)
+        const park = await DogPark.query().findById(id)
+        const serializedPark = await DogParksSerializer.detailsForShow(park)
+        res.set({"Content-Type": "application/json"}).status(201).json({ park: serializedPark })
+    } catch(err) {
+        if (err instanceof ValidationError) {
+            res.set({"Content-Type": "application/json"}).status(422).json({ errors: err.data })
+        } else {
+            res.set({"Content-Type": "application/json"}).status(500).json({ errors: err })
```

Comment:
setting content type is not needed - consider consistency with the other router endpoints in your app